docs/features/task-types.md

```diff
@@ -25,12 +25,12 @@ class Awaiter<T> : INotifyCompletion
 }
 ```
 ## Builder Type
-The _builder type_ is a `class` or `struct` that corresponds to the specific _task type_.
+The _builder type_ is a public `class` or `struct` that corresponds to the specific _task type_.
```

Comment:
I don't think that's correct. From looking at the code (`ValidateBuilderType`) we're requiring for the builder type  to have the same accessibility as the return type.

Reply:
Yes, it looks like the builder type should have the same accessibility as the task-like type.

Reply:
Sorry then, I will close the PR. Thanks for your time!